src/lib/stores/account.test.ts

```diff
@@ -0,0 +1,62 @@
+import { zBechAddr } from "lib/types";
+
+import { AccountStore } from "./account";
+
+describe("accountStore", () => {
+  let accountStore: AccountStore;
+
+  beforeEach(() => {
+    accountStore = new AccountStore();
+  });
+
+  test("user key management", () => {
+    expect(accountStore.isAccountUserKeyExist()).toBeFalsy();
+
+    accountStore.setAccountUserKey("user-key");
+
+    expect(accountStore.isAccountUserKeyExist()).toBeTruthy();
+  });
+
+  test("account info", () => {
+    expect(
+      accountStore.getAccountLocalInfo(zBechAddr.parse("address"))
```

Comment:
Make variable for this `zBechAddr.parse("address")`

Reply:
👍🏼